zulipterminal/core.py

```diff
@@ -294,7 +294,7 @@ def show_stream_members(self, stream_id: int) -> None:
         stream_members_view = StreamMembersView(self, stream_id)
         self.show_pop_up(stream_members_view, "area:stream")
 
-    def popup_with_message(self, text: str, width: int) -> None:
+    def show_popup_with_message(self, text: str, width: int) -> None:
```

Comment:
Got it! That makes sense. I couldn't come up with a good substitute for the word "popup" either, but since this is internal code, I thought it's fine to leave as is.